internal/server/statvar/statvar_search.go

```diff
@@ -234,15 +238,18 @@ func searchTokens(
 		}
 	}
 
+	matchingStrings := map[string]struct{}{}
```

Comment:
since this isn't entirely straightforward, worth a comment that this is a set, and that struct{}{} is an empty struct.

you could also use `var exists = struct{}{}` to help readability

https://www.davidkaya.com/sets-in-golang/

Reply:
done